src/components/CloneCom/CloneCom.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { useUserContext } from '../../context/UserContext';
+
+export default function CloneCom({ note, handleClone }) {
+  //   const { submitClone, setTitle, setData, title } = Clone();
+  const [title, setTitle] = useState(note.title);
+  const [data, setData] = useState(note.note);
```

Comment:
Consider using NoteContext for managing state instead of calling state directly here.